LethalAPI.Core/MEC/Timing.cs

```diff
@@ -174,13 +177,18 @@ public static Timing Instance
             {
                 if (_instance == null || !_instance.gameObject)
                 {
+                    Log.Error("Timings is not initialized! You cannot use timings until the player has reached the menu.");
+                    throw new Exception("Timings is not initialized! You cannot use timings until the player has reached the menu.");
                     GameObject instanceHome = GameObject.Find("Timing Controller");
 
                     if (instanceHome == null)
                     {
-                        instanceHome = new GameObject { name = "Timing Controller" };
+                         //instanceHome = GameObject.FindGameObjectWithTag("BepInEx_Manager");
+                         instanceHome = GameObject.FindObjectsOfType<GameObject>()
+                             .First(x => x.name == "BepInEx_Manager");
+                        // instanceHome = new GameObject { name = "Timing Controller" };
 
-                        DontDestroyOnLoad(instanceHome);
+                        //DontDestroyOnLoad(instanceHome);
```

Comment:
Why are these commented-out lines still in? Is there any reason we don't remove them?

Reply:
It should be unnecessary now. MEC is designed for being added into the game pre-compilation, which means unity abstracts this and changes / removes many keywords and methods before compilation begins. I believe that this was a result of that. It works now so I'm content on removing it.